src/codecatalyst/auth.ts

```diff
@@ -94,6 +94,23 @@ export class CodeCatalystAuthenticationProvider {
         await this.secondaryAuth.restoreConnection()
     }
 
+    public async promptOnboarding(): Promise<void> {
+        const okItem: vscode.MessageItem = { title: localizedText.ok }
+        const cancelItem: vscode.MessageItem = { title: localizedText.cancel, isCloseAffordance: true }
+        const resp = await vscode.window.showInformationMessage(
+            'The current connection is not onboarded with CodeCatalyst. This must be done in the browser.\n\n Onboard now?',
```

Comment:
- "onboard" is somewhat jargon, should we say "sign up" instead? Or is CC using "onboard" in their materials?
- "This must be done in the browser" is a lot of words for a rather common task, perhaps we can find tighter verbiage:

```suggestion
            'The current connection is not onboarded with CodeCatalyst. Visit sign-up page now?',
```

vscode also supports hyperlinks now (and this dialog isn't needed in Cloud9), so we could do something like:

    [Sign up to CodeCatalyst](https://...), to get started.

Reply:
This works although the notification doesn't show fully expanded which is kind of annoying.